examples/cli/src/test/java/io/unitycatalog/cli/table/CliExternalTableCreationTest.java

```diff
@@ -0,0 +1,155 @@
+
+public class CliExternalTableCreationTest extends BaseServerTest {
+
+
+    private CatalogOperations catalogOperations;
+    private SchemaOperations schemaOperations;
+    private TableOperations tableOperations;
+
+    private static final String SCHEMA_FULL_NAME = CATALOG_NAME + "." + SCHEMA_NAME;
+    private static final String TABLE_FULL_NAME = CATALOG_NAME + "." + SCHEMA_NAME + "." + TABLE_NAME;
+
+    @Before
+    public void setUp() {
+        super.setUp();
+        catalogOperations = new CliCatalogOperations(serverConfig);
+        schemaOperations = new CliSchemaOperations(serverConfig);
+        tableOperations = new CliTableOperations(serverConfig);
+        try {
+            catalogOperations.createCatalog(CATALOG_NAME, "Test catalog");
+            schemaOperations.createSchema(new CreateSchema().name(SCHEMA_NAME).catalogName(CATALOG_NAME).comment("Test schema"));
+        } catch (Exception e) {
+            // Ignore
+        }
+    }
+
+    @Override
+    public void cleanUp() {
+
+        try {
+            if (tableOperations.getTable(TABLE_FULL_NAME) != null) {
+                tableOperations.deleteTable(TABLE_FULL_NAME);
+            }
+        } catch (Exception e) {
+            // Ignore
+        }
+        try {
+            if (schemaOperations.getSchema(SCHEMA_FULL_NAME) != null) {
+                schemaOperations.deleteSchema(SCHEMA_FULL_NAME);
+            }
+        } catch (Exception e) {
+            // Ignore
+        }
+        try {
+            if (catalogOperations.getCatalog(CATALOG_NAME) != null) {
+                catalogOperations.deleteCatalog(CATALOG_NAME);
+            }
+        } catch (Exception e) {
+            // Ignore
+        }
+    }
+
+    @AfterEach
+    public void perTestCleanup() throws ApiException {
+        try {
+            if (tableOperations.getTable(TABLE_FULL_NAME) != null) {
+                tableOperations.deleteTable(TABLE_FULL_NAME);
+            }
+        } catch (Exception e) {
+            // Ignore
+        }
+    }
+
+    @Test
+    public void testCreateTableLocalDirectoryDoesNotExist() throws IOException, ApiException {
+        String tablePath = "/tmp/" + UUID.randomUUID();
+        TableInfo tableInfo = tableOperations.createTable(new CreateTable().catalogName(CATALOG_NAME).schemaName(SCHEMA_NAME).name(TABLE_NAME).storageLocation(tablePath).columns(
+                List.of(
+                        new ColumnInfo().name("id").typeName(ColumnTypeName.INT).comment("id"),
+                        new ColumnInfo().name("name").typeName(ColumnTypeName.STRING).comment("name"))));
+        assertNotNull(tableInfo);
+        assertNotNull(tableInfo.getTableId());
+        assertEquals(tableInfo.getTableId(), tableOperations.getTable(TABLE_FULL_NAME).getTableId());
+        assertDoesNotThrow(() -> DeltaKernelUtils.readDeltaTable(tableOperations.getTable(TABLE_FULL_NAME).getStorageLocation(), null, 100));
+        deleteDirectory(Paths.get(tablePath));
```

Comment:
Nit: refactor this code (it is duplicated in all the three tests).

Reply:
Moved common logic to a function.